### DISCUSSION TITLE
Unable to optimize any image in Static folder without using "require"

### DISCUSSION THREAD

**Discussion by ManasMadrecha** [juliomrqz/nuxt-optimized-images#301] [Q&A]
I am using Nuxt Content. So, to avoid rebuilding every time I add new images or post, I have all my images in Static folder (arranged blog post-wise).

So, in my Individual blog post page, for my Featured image:

I do not use 
`<img :src="require('~/assets/image.jpg)?webp" />`

Rather, I use
``<img :src="`/images/blog/${post.id}/${post.featured.image}.jpg?webp" />``

Then, when I `nuxt generate`, I am unable to see any optimization. Even in my image URL, `?webp` stays in the end, which means nothing happened to the image.

Is it because this module only optimizes images in **Assets** folder and not in **Static** folder?

**Comment by dikaio** [comment]
Also interested in this answer @ManasMadrecha. Curious have you also tried using this module in a standard page component and not via content module? Also, by design files that will not change would stay in the static folder and files that will be modified in some capacity e.g. nuxt-optimized-images would go in the assets folder.

There is an open issue now being discussed about processing images within the Content Module [here](https://github.com/nuxt/content/issues/106).

**Reply by dikaio** [reply]
@juliomrqz curious do you have any example implementations of the `nuxt-optimized-images` and the [Content Module](https://content.nuxtjs.org)?

As soon as [issue 106](https://github.com/nuxt/content/issues/106) is worked out I'd be happy to work on a pull request if you're okay with it and if it's needed.
